e2e-tests/cypress/cypress.config.ts

```diff
@@ -14,9 +14,9 @@ export default defineConfig({
     video: false,
     viewportWidth: 1300,
     env: {
-        adminEmail: 'sysadmin@sample.mattermost.com',
-        adminUsername: 'sysadmin',
-        adminPassword: 'Sys@dmin-sample1',
+        adminEmail: process.env.CY_ADMIN_EMAIL || 'sysadmin@sample.mattermost.com',
+        adminUsername: process.env.CY_ADMIN_USERNAME || 'sysadmin',
+        adminPassword: process.env.CY_ADMIN_PASSWORD || 'Sys@dmin-sample1',
```

Comment:
This is fine but Cypress allows overriding these config options via env variable, like `CYPRESS_adminUsername`. We have it documented as well - https://developers.mattermost.com/contribute/more-info/webapp/e2e-testing/#environment-variables.
Let me know if that works.

Reply:
Oh, that's perfect. I didn't realize that we had those, and I didn't think to check in the docs for them since we didn't have anything explicit for them in the code like we do in the Playwright config. I'll close this then